plugin/tiff/src/test/java/it/geosolutions/imageio/tiff/TIFFWriteTest.java

```diff
@@ -263,17 +263,13 @@ public void imageComplete(ImageWriter source) {
         reader.dispose();
     }
 
-
-
     @Test
     public void writeZSTD() throws IOException {
-

```

Comment:
Only whitespace changes in this file, it seems.

Reply:
Ops... a rebase issue, I think. fixing